### ISSUE TITLE
Multiple muxing some files results in files 36 bytes larger.

### ISSUE THREAD

**Issue by jsmulree** [mbunkus/mkvtoolnix#1503]
Hello mbunkus,

I have noticed that muxing some files multiple times results in a file that is 36 bytes larger after each muxing. This issue does not affect all files and I have determined that this issue appears to only affect certain BluRay rips and is easily reproduced - I have tried several different files and have successfully reproduced the issue for most of them.

Steps to reproduce:
1) Produce an initial MKV file within MKVToolNix, configuring any options as appropriate. File size is 'x' bytes.
2) Mux the file from step '1' using MKVToolNix without changing any of the default options (except perhaps the output file). If affected by this issue, the resulting file will be 'x+36' bytes.
3) If the file from step '2' is affected by this issue, mux this file using MKVToolNix without changing any of the default options. The resulting file will be 'x+36+36' bytes.

This issue does not appear to negatively impact the usability of an affected file and is really an annoyance more than anything else. I am reporting this issue just in case an affected file is negatively impacted in a way that is not immediately obvious.

Cheers,

Jason.

**Comment by mbunkus** [comment]
Thanks for the report. Can you please provide a sample file with which to reproduce the issue? You can upload it to [my FTP server](/mbunkus/mkvtoolnix/wiki/FTP-server). Thanks.

**Comment by jsmulree** [comment]
Hello mbunkus,

I have uploaded the sample file as requested ('jsmulree (#1503).7z'), including the output of the multiple muxings. Within the compressed file, the numbers  as denoted by the number in brackets. The sample 'jsmulree (0)' was produced by muxing a larger movie (that reproduced the issue) using MKVToolNix to generate a 10 second split of the file. I then followed the steps outlined earlier to produce the subsequent files. Interestingly this yielded different slightly different results to those I have previously encountered, though the specific issue reported emerges from file 'jsmulree (3)'.

I hope this helps! :-)

Cheers,

Jason.
